docs/rst/installation/linux_installation.rst

```diff
@@ -2,3 +2,325 @@
+        The following setup call must be run in order to set the *CLI* command:
+
+        .. code-block:: bash
+
+            export LD_LIBRARY_PATH=~/Fast-DDS-QoS-Profiles-Manager/install/lib/lib
+            export PATH=$PATH:~/Fast-DDS-QoS-Profiles-Manager/install/cli/bin
+
+
+.. _gui_installation_sl:
+
+Fast DDS QoS Profiles Manager GUI
+---------------------------------
+
+.. warning::
+
+    This section is still in development process.
+
+.. _doc_installation_sl:
+
+Fast DDS QoS Profiles Manager Documentation
+-------------------------------------------
+
+Finally, this section describes the instructions for installing *eProsima Fast DDS QoS Profiles Manager documentation*
+in a Linux environment from sources.
+
+The *eProsima Fast DDS QoS Profiles Manager library* is required to build the *documentation*, so make sure all steps
+described in the :ref:`library installation section <library_api_installation_sl>` were reproduced.
+Despite that, :ref:`doc_dependencies_sl` detailed below need to be met.
+
+.. _doc_dependencies_sl:
+
+Dependencies
+^^^^^^^^^^^^
+
+*eProsima Fast DDS QoS Profiles Manager documentation* depends on `Doxygen code documentation generator
+<https://www.doxygen.nl/>`_ for the :ref:`fastdds_qos_profiles_manager_lib_api_reference`.
+In Ubuntu, the dependency can be installed running:
+
+.. code-block:: bash
+
+    sudo apt install doxygen
+
+.. note::
+
+    If desired, the remain *documentation* dependencies can be installed in a python3 virtual environment.
+    That would avoid polluting the user's installation.
+    Create the virtual environment by running the following commands:
+
+    .. code-block:: bash
+
+        sudo apt install python3
+        cd ~/Fast-DDS-QoS-Profiles-Manager
+        python3 -m venv fastdds_qos_profiles_manager_venv
+        source fastdds_qos_profiles_manager_venv/bin/activate
+
+Install the dependencies using `pip3 <https://docs.python.org/3/installing/index.html>`_:
+
+.. code-block:: bash
+
+    cd ~/Fast-DDS-QoS-Profiles-Manager/src/Fast-DDS-QoS-Profiles-Manager/docs
+    pip3 install -r requirements.txt
+
+
+.. _doc_building_sl:
+
+Build
+^^^^^
+
+.. tabs::
+
+    .. tab:: Building with colcon
+
+        *eProsima Fast DDS QoS Profiles Manager documentation* is build using
+        `colcon <https://colcon.readthedocs.io/en/released/>`_ by running:
+
+        .. code-block:: bash
+
+            cd ~/Fast-DDS-QoS-Profiles-Manager
+            colcon build --packages-select fastdds_qos_profiles_manager_lib fastdds_qos_profiles_manager_docs
+
+    .. tab:: Building with CMake
+
+        *eProsima Fast DDS QoS Profiles Manager documentation* is build using `CMake <https://cmake.org/>`_ by running:
+
+        .. code-block:: bash
+
+            mkdir -p ~/Fast-DDS-QoS-Profiles-Manager/build/docs
+            cd ~/Fast-DDS-QoS-Profiles-Manager/build/docs
+            CMAKE_PREFIX_PATH+=~/Fast-DDS-QoS-Profiles-Manager/install/lib
```

Comment:
Please, apply the same suggestion. Although, as it is being run in the same snippet, this step should be done only once, but I suggest adding a note explaining that this is required so the library dependency is found.